RELEASING.md

```diff
@@ -88,45 +91,48 @@ In order to keep this relevant & accurate, we improve this doc during the
 release process. It's the best time to pause, observe how it all fits together,
 and improve it. We want small, constant improvements which compound. Therefore:
 
-- [ ] Open this doc in your code editor. As you go through these steps, check
+- [x] Open this doc in your code editor. As you go through these steps, check
```

Comment:
Did you mean to leave all the checkboxes pre-checked? No biggie either way, just checking